### PR TITLE
adding configuration text color in dialog permission

### PR DESCRIPTION
**Pull request by adicatur**
User able to config dialog permission text color :
1. For positive button :
```
Qiscus.getChatConfig()
                .setDialogPermissionPositiveButtonTextColor(R.color.primary);
```
2. For negative button :
```
Qiscus.getChatConfig()
                .setDialogPermissionNegativeButtonTextColor(R,color.acent);
```

### CODE REVIEW

File: chat/src/main/java/com/qiscus/sdk/util/QiscusPermissionsUtil.java [hunk 1]
```diff
@@ -106,6 +109,14 @@ public static void requestPermissions(final Object object, String rationale,
                         callbacks.onPermissionsDenied(requestCode, Arrays.asList(perms));
                     }).create();
             dialog.show();
+
+            dialog.getButton(DialogInterface.BUTTON_POSITIVE).setTextColor
```
Comment (zetbaitsu) [inline chat/src/main/java/com/qiscus/sdk/util/QiscusPermissionsUtil.java:113]: di taroh sebelum di show aja ya pak @adicatur 
sebelum ```dialog.show();```

File: chat/src/main/java/com/qiscus/sdk/util/QiscusPermissionsUtil.java [hunk 2]
```diff
@@ -169,8 +180,16 @@ public static boolean checkDeniedPermissionsNeverAskAgain(Object object, String
                         })
                         .setNegativeButton(negativeButton, null)
                         .create();
+
                 dialog.show();
 
+                dialog.getButton(DialogInterface.BUTTON_POSITIVE).setTextColor
```
Comment (zetbaitsu) [inline chat/src/main/java/com/qiscus/sdk/util/QiscusPermissionsUtil.java:186]: sama ya sebelum ```dialog.show();```

**Review by zetbaitsu — CHANGES_REQUESTED** [review]

File: chat/src/main/java/com/qiscus/sdk/util/QiscusPermissionsUtil.java (hunk 2 above)
Comment (adicatur) [reply chat/src/main/java/com/qiscus/sdk/util/QiscusPermissionsUtil.java:186]: kalau sebelm show null pak

File: chat/src/main/java/com/qiscus/sdk/util/QiscusPermissionsUtil.java (hunk 1 above)
Comment (adicatur) [reply chat/src/main/java/com/qiscus/sdk/util/QiscusPermissionsUtil.java:113]: kalau sebelm show null pak

Comment (zetbaitsu) [reply chat/src/main/java/com/qiscus/sdk/util/QiscusPermissionsUtil.java:113]: Wew nice :v

**Review by zetbaitsu — APPROVED** [review]

**Pull request merged**